test/Spring-test.js

```diff
@@ -167,26 +258,64 @@ describe('Spring', () => {
       },
     });
     TestUtils.renderIntoDocument(<App />);
+
+    // Initial render
+    expect(count).toEqual([0]);
+    mockRaf.manySteps(8);
+    expect(count).toEqual([
+      0,
```

Comment:
One list doesn't cover much though? I guess it doesn't matter here since if you get one list right you'd get others right